jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererCodeScanManager.kt

```diff
@@ -387,6 +397,22 @@ class CodeWhispererCodeScanManager(val project: Project) {
         project.refreshDevToolTree()
     }
 
+    private fun sendCodeScanTelemetryToServiceAPI(
+        project: Project,
+        programmingLanguage: CodeWhispererProgrammingLanguage?,
+        codeScanJobId: String?
+    ) {
+        checkIfIamIdentityCenterConnection(project) {
+            try {
+                val response = CodeWhispererClientAdaptor.getInstance(project)
+                    .sendCodeScanTelemetry(programmingLanguage, codeScanJobId)
+                LOG.debug { "Successfully sent code scan telemetry. RequestId: ${response.responseMetadata().requestId()}" }
+            } catch (e: Exception) {
+                LOG.error(e) { "Failed to send code scan telemetry." }
```

Comment:
is `LOG.error` on purpose? asking because `error` level will pop up at the IDE bottom right corner and customer will see it. `Debug` should make more sense here?

Reply:
I will change to debug,